addon/components/node-tree/tree/index.js

```diff
@@ -28,6 +30,7 @@ export default class NodeTreeComponent extends Component {
   filterNodesFn;
 
   transition = verticalSlide;
+  selectedNodes = new TrackedWeakSet();
```

Comment:
Yeahp, already using tracked-built-ins on my more recent projects, thanks for updating it. As for the `set` usage, this was an MPV that was only meant to deal with ember-data (ember-objects) models to kick off reactiveness at the time. Not sure how might this work with ember-data, but it'll do awesome with anything else 👍🏻